.github/workflows/ci.yaml

```diff
@@ -102,6 +106,8 @@ jobs:
         with:
           name: release-packages
           path: ./release-packages
+      - name: Remove docker images
+        run: docker system prune -af
```

Comment:
Just curious why it needs to run twice - not saying it's wrong or anything, but maybe a comment would help

Reply:
It shouldn't! (I hope not at least). But I think I need to run it at the beginning once to fix the issue, then I can remove that line and I'll keep the one that runs at the end. Don't know if there's an easier way to do it - just the first one that came to mind 😂

Reply:
makes  sense! ty!